.github/workflows/docker.yml

```
@@ -0,0 +1,33 @@
+name: docker
```

```suggestion
name: Build registry
```

So it doesn't get confused with Agora's docker job. Same for the file name, something like `registry-docker.yml` or whatever you see fit.

---

I planned to put both agora and name-registry jobs into the same workflow file, so if one fails, then we don't attempt to build the other. Anyways, let me separate them.